core-api/src/main/java/com/sequenceiq/cloudbreak/api/endpoint/v4/stacks/response/database/DatabaseServerStatus.java

```diff
@@ -27,5 +30,16 @@ public enum DatabaseServerStatus {
     UPGRADE_REQUESTED,
     UPGRADE_IN_PROGRESS,
     UPGRADE_FAILED,
-    UNKNOWN
+    UNKNOWN;
+
+    private static final List<DatabaseServerStatus> IS_AVAILABLE_LIST = Arrays.asList(AVAILABLE, MAINTENANCE_MODE_ENABLED);
```

Comment:
Maintenance mode in Redbeams is not really used, only in redbeams-api. I would not add it to the available list.